styles/form/forms.css

```diff
@@ -1,7 +1,7 @@
 input {
   padding: 0.5rem 1rem;
   font-size: 1rem;
-  border: 2px solid gray;
+  border: var(--space-1) solid gray;
```

Comment:
You could be consistent and only use variables. If this gray is only used in this component, you can create a variable only for this component.

Reply:
oh..yeah...could've added that.

Reply:
done have added that! Thanks for the review.